### ISSUE TITLE
loss fluctuates when training on a custom dataset, is this normal behaviour in contrastive learning?

### ISSUE THREAD

**Issue by rahzaazhar** [kavehhassani/mvgrl#4]

**Comment by kavehhassani** [comment]
Have you fine-tuned your hyper-params? There is a bit of fluctuation but it is not that much.
